diff --git a/binsrc/sqldoc/isql_log_normalize.c b/binsrc/sqldoc/isql_log_normalize.c
index cb54667..93f7473 100644
--- a/binsrc/sqldoc/isql_log_normalize.c
+++ b/binsrc/sqldoc/isql_log_normalize.c
@@ -56,8 +56,6 @@ typedef struct cut_env_s
 
 static char errmsg_buf[1000];
 
-char *strdup(const char *s);
-
 void readtextfile (const char *src_name, char **bufptr, size_t *lenptr, char **err_ret)
 {
   FILE *f = fopen (src_name, "rt");
